src/js/injection_script.js

```diff
@@ -2325,6 +2325,7 @@ ensureDomLoaded(()=>{
 	})
 	domainBypass("techynroll.com", ()=>awaitElement("a#enablebtn", a=>safelyAssign(a.href)))
 	hrefBypass(/meostream\.com\/links\//,()=> ifElement("a#link",safelyNavigate))
+	domainBypass("metroupdate.biz", ()=>myFunction())
```

Comment:
Ensure that the DOM has been loaded first.
```suggestion
	domainBypass("metroupdate.biz", ()=>ensureDomLoaded(()=>myFunction()))
```

Reply:
It is already in ensureDomLoaded
	//Insertion point for domain-or-href-specific bypasses running after the DOM is loaded. Bypasses here will no longer need to call ensureDomLoaded.

Reply:
👍